packages/@webex/plugin-meetings/src/meeting-info/index.ts

```diff
@@ -113,21 +141,39 @@ export default class MeetingInfo {
    * @public
    * @memberof MeetingInfo
    */
-  public fetchMeetingInfo(destination: string, type: string = null) {
+  public fetchMeetingInfo(
+    destination: string,
+    type: string = null,
+    // eslint-disable-next-line @typescript-eslint/no-unused-vars
+    password: string = null,
+    // eslint-disable-next-line @typescript-eslint/no-unused-vars
+    captchaInfo: {
+      code: string;
+      id: string;
+    } = null,
+    // eslint-disable-next-line @typescript-eslint/no-unused-vars
+    installedOrgID = null,
+    // eslint-disable-next-line @typescript-eslint/no-unused-vars
+    locusId = null,
+    // eslint-disable-next-line @typescript-eslint/no-unused-vars
+    extraParams: object = {},
+    options: {meetingId?: string} = {}
+  ) {
     if (type === _PERSONAL_ROOM_ && !destination) {
       destination = this.webex.internal.device.userId;
     }
 
     return this.fetchInfoOptions(MeetingInfoUtil.extractDestination(destination, type), type).then(
-      (options) =>
+      (infoOptions) =>
         // fetch meeting info
         this.requestFetchInfo(options).catch((error) => {
```

Comment:
before this change the call to `this.requestFetchInfo()` was passing the options that were the result of the call to `this.fetchInfoOptions()`, now we're passing the options which are just the `meetingId` - this doesn't seem right, we should pass `{...infoOptions, meetingId: options.meetingId}`, right?

BTW, I would expect that there is some existing unit or integration test that should catch this. Was it caught by any test?

Reply:
oh, i missed that there are 2 calls to this.requestFetchInfo, will add it to the first call as well.
I don't think it was caught, I have a green build..

Reply:
but there were no tests for meeting info v1, i had to create a new file, but only tested the new changes i introduced

Reply:
fixed it and added a test